Proto/cg.bat

```diff
@@ -0,0 +1,7 @@
+@echo off
```

Comment:
Yes, it might be a good time to start a "no more bat files" rule :)  (at least for everything apart from the ``vsintegration`` directory)